content/projects/katas/level-1/_index.md

```diff
@@ -61,6 +61,9 @@ If you are writing JS code then your files would follow a similar pattern. You w
 ```js
 module.exports = { YOUR_FUNCTION_NAME };
 ```
+## Note to Java developers:
+
+- Even though the instructions will specifically ask for a function, please don't submit only the function/method. Make sure that there is a wrapping public class in your code.
```

Comment:
Please add examples. 

The learners will either put everything in the same class or spread all the functions out across multiple files.  The learners will end up guessing.  It's important to be very very explicit about what is required here. 

Please reach out to @ruddy-m  to ask her if you aren't sure how the code should be structured.